test/gov/multiple_agents.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+COUNTER=0
+if [ ${MUL_AGENTS} -lt 5 ]; then
+  NUM_AGENTS=${MUL_AGENTS}
+else
+  NUM_AGENTS=4
+fi
+while [ ${COUNTER} -lt ${NUM_AGENTS} ]; do
+  echo ${COUNTER}
+  AGENT_PORT=$((8512 + ${COUNTER}))
+  DEVICE_NUM=$((6 + ${COUNTER}))
+  echo ${AGENT_PORT}
+  echo ${DEVICE_NUM}
```

Comment:
Instead of these 3 echos filling up the console, it might be better to just echo the generated /tmp/horizon file after you've completed it.